shared/actions/login/index.js

```diff
@@ -392,7 +392,16 @@ const makeKex2IncomingMap = (dispatch, getState, onBack: SimpleCB, onProvisioner
     'keybase.1.provisionUi.DisplayAndPromptSecret': ({phrase, secret}, response) => {
       dispatch({payload: {textCode: new HiddenString(phrase)}, type: Constants.setTextCode})
       generateQRCode(dispatch, getState)
-      dispatch(askForCodePage(phrase => { response.result({phrase, secret: null}) }, () => onBack(response)))
+      let codeSent = false
```

Comment:
could also just set response to null and check it before sending, don't need the extra var